lang/funcs/core/coresys/cpucount_test.go

```diff
@@ -0,0 +1,66 @@
+// Mgmt
```

Comment:
This file name should be _test.go added to the one we're testing. So we want it to be named cpucount_fact_test.go instead.